### DISCUSSION TITLE
怎么将默认粘贴（快捷键+右键粘贴）改成只粘贴值+粘贴格式

### DISCUSSION THREAD

**Discussion by keandfan** [dream-num/univer#3079] [Q&A | 常见问题]
https://univer.ai/zh-CN/guides/sheet/advanced/custom-command#%E5%9C%A8%E5%85%B6%E4%BB%96-command-%E6%89%A7%E8%A1%8C%E7%9A%84%E6%97%B6%E6%9C%BA%E8%A1%A5%E5%85%85-mutations
官方的例子我看了，好像看不太懂，能否出一个plaground的，或者更新一下官方例子

**Comment by jikkai** [comment]
All example codes need to be maintained, and considering the issue of manpower shortage, we will not provide more examples for specific businesses. 

If there are parts you do not understand, please specify what they are. Your description does not allow us to further assist you.

**Reply by keandfan** [reply]
https://stackblitz.com/edit/awesome-univer-sheets-vue3-demo-ttnufu?file=plugins%2Fcontroller%2Findex.ts,plugins%2Findex.ts


我按官网的例子来操作，到这一步就不知道怎么做了，我的目的是
1.粘贴的时候  可以粘贴值 + 粘贴格式，其他的不要

希望能得到一些帮助，实在不知道怎么做了。 
![image](https://github.com/user-attachments/assets/994c5e3b-047d-486c-b438-e9d31ebba919)
默认粘贴的效果就是上面这些就行了，redo undo 看官网例子不知道怎么操作的。。 希望得到您的帮助
